definitions/grib2/marsLevtypeConcept.def

```diff
@@ -13,13 +13,15 @@
 'pl'   = {typeOfFirstFixedSurface=100; typeOfSecondFixedSurface=255;}
 'pl'   = {typeOfFirstFixedSurface=100; typeOfSecondFixedSurface=100;}
 'sfc'  = {typeOfFirstFixedSurface=101; typeOfSecondFixedSurface=255;}
-'sfc'  = {typeOfFirstFixedSurface=102; typeOfSecondFixedSurface=255;}
+# heightAboveSea
+'ml'   = {typeOfFirstFixedSurface=102; typeOfSecondFixedSurface=255;}
 'sfc'  = {typeOfFirstFixedSurface=103; typeOfSecondFixedSurface=255;}
 'sfc'  = {typeOfFirstFixedSurface=103; typeOfSecondFixedSurface=103;}
 'ml'   = {typeOfFirstFixedSurface=105; typeOfSecondFixedSurface=255;}
 'ml'   = {typeOfFirstFixedSurface=105; typeOfSecondFixedSurface=105;}
-'sfc'  = {typeOfFirstFixedSurface=106; typeOfSecondFixedSurface=255;}
-'sfc'  = {typeOfFirstFixedSurface=106;typeOfSecondFixedSurface=106;}
+# depthBelowLand
+'ml'   = {typeOfFirstFixedSurface=106; typeOfSecondFixedSurface=255;}
+'ml'   = {typeOfFirstFixedSurface=106;typeOfSecondFixedSurface=106;}
 'pt'   = {typeOfFirstFixedSurface=107; typeOfSecondFixedSurface=255;}
```

Comment:
We should define a special value for soil levels here, e.g. 'sl' or something more expressive. Can discuss that with Sebastien, when we review our model.I still have to figure out how to solve the scaling issue.